bench/Bench/Data/Map.purs

```diff
@@ -42,6 +42,12 @@ benchMap = do
   log "---------------"
   benchValues
 
+  log ""
+
+  log "keys"
+  log "---------------"
+  benchValues
```

Comment:
This should be benchKeys

Reply:
Sorry about that. I'll fix and repost benchmarks.

Reply:
Opening comment now has updated benchmarks.